src/main.py

```diff
@@ -151,7 +200,11 @@ def matricule(frame, outs, width_factor=1.1, height_factor=1.0):
     except Exception as e:
         print("Erreur lors de la détection de la plaque d'immatriculation:", e)
 
+
 class VehicleCounter:

```

Comment:
Think of adding a cleanup function to release resources such as OpenCV video captures, something like :
            cam.release()  # Release video capture
            cv2.destroyAllWindows()  # Close all OpenCV windows